WordPress/Classes/ViewRelated/Blog/EditSiteViewController.m

```diff
@@ -16,11 +30,35 @@
 #import "AccountService.h"
 #import "ContextManager.h"
 
+const typedef enum {
+    EditSiteRowURL = 0,
+    EditSiteRowUsername = 1,
+    EditSiteRowPassword = 2,
+    EditSiteRowGeotagging = 0,
+    EditSiteRowPushNotifications = 1,
+    EditSiteRowJetpackConfigure = 0
+} EditSiteRow;
+
+const typedef enum {
+    TableViewSectionTitle = 0,
+    TableViewSectionSettings,
+    TableViewSectionSelfHosted
+} TableSectionContentType;
+
 static NSString *const TextFieldCellIdentifier = @"TextFieldCellIdentifier";
 static NSString *const GeotaggingCellIdentifier = @"GeotaggingCellIdentifier";
 static NSString *const PushNotificationsCellIdentifier = @"PushNotificationsCellIdentifier";
 static NSString *const JetpackConnectedCellIdentifier = @"JetpackConnectedCellIdentifier";
 static CGFloat const EditSiteRowHeight = 48.0;
+NSInteger const URLMinimumLabelWidth = 30;
```

Comment:
Just to keep with the naming scheme for the other constants, why not rename this to `EditSiteURLMinimumLabelWidth`?  I know it's longer but it's a good way to avoid a name clash, and to make it clear it's part of the local set of static variables.